src/test/scala/io/iohk/ethereum/blockchain/sync/ScenarioSetup.scala

```diff
@@ -72,56 +74,18 @@ trait ScenarioSetup extends StdTestConsensusBuilder with SyncConfigBuilder with
   protected def newTestConsensus(validators: Validators = consensus.validators, vm: VMImpl = consensus.vm): Consensus =
     consensus.withValidators(validators).withVM(vm)
 
-  /**
-    * Creates a new ledger instance, independent of the instance provided by the cake.
-    *
-    * @note Since ledger depends on the consensus, it is the caller's responsibility to
-    *       make sure that the cake-provided consensus and the one used here do not interfere.
-    *       In particular, after the new ledger is created, then the authoritative consensus for the ledger
-    *       is the one returned by [[io.iohk.ethereum.ledger.Ledger#consensus() Ledger#consensus]], not the
-    *       one provided by the cake.
-    *
-    * @note You can use this method to override the existing ledger instance that is provided by the cake.
-    */
-  protected def newTestLedger(consensus: Consensus): LedgerImpl =
-    new LedgerImpl(
-      blockchain = blockchain,
-      blockchainConfig = blockchainConfig,
-      syncConfig = syncConfig,
-      theConsensus = consensus,
-      validationContext = monixScheduler
-    )
-
-  protected def newTestLedger(blockchain: BlockchainImpl): LedgerImpl =
-    new LedgerImpl(
-      blockchain = blockchain,
-      blockchainConfig = blockchainConfig,
-      syncConfig = syncConfig,
-      theConsensus = consensus,
-      validationContext = monixScheduler
+  protected def mkBlockImport(validators: Validators = validators, blockExecutionOpt: Option[BlockExecution] = None) = {
+    val consensuz = consensus.withValidators(validators).withVM(new Mocks.MockVM())
```

Comment:
```suggestion
    val consensus = consensus.withValidators(validators).withVM(new Mocks.MockVM())
```

Reply:
As before; don't want a recursive definition (and make it explicit that it's a different one.)